tests/test_integration.py

```diff
@@ -455,6 +456,21 @@ def test_info(self):
         self.vm.shutdown()
         host_info(VM_HOSTNAME)
 
+    @patch('igvm.vm.VM.vm_performance_value', return_value=5.0)
+    def test_igvm_migration_log(self, mock_vm_performance_value):
+        regex = r'\A([0-9]{10}\s[-|+][0-9]{1,3})+\Z'
+
+        for hv in self.hvs:
+            if hv.dataset_obj['hostname'] == self.vm.dataset_obj['hypervisor']:
+                hv.dataset_obj['igvm_migration_log'] = ''
```

Comment:
igvm_migration_log is a multi attribute which implements the default python set interface. While it is supported to set it to an empty string to remove all values it is nicer to use the clear() method to make the reader aware that this a multi attribute.

Reply:
Using clear() now.